src/java.base/share/classes/java/lang/Object.java

```diff
@@ -62,7 +63,7 @@
      * Constructs a new object.
      */
     @HotSpotIntrinsicCandidate
-    public Object() {}
+    public @Untainted Object() {}
```

Comment:
Should you also add tainting to `@AnnotatedFor`?

Reply:
No, all the methods have not been annotated, just this one.

Reply:
Our current suggestion is to fully annotate files: https://checkerframework.org/manual/#annotating-jdk
j.l.Object isn't very big, so wouldn't it be better to fully annotate it for the Tainting Checker?
The "Whenever you add a file" part no longer makes sense, as all possible files are already contained in the typetools/jdk repository.

Reply:
We will do that later.

Reply:
You should at least open an issue to keep track of this.